portal/app/core/templates/resources.html

```diff
@@ -35,7 +35,7 @@
                             </a>
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link text-center icons w3-bar-item w3-button" href="/">
+                            <a class="nav-link text-center icons w3-bar-item w3-button {% if resource == 'battery' %}current{% endif %}" href="/battery">
```

Comment:
makes this a triple === and then the PR should be gtg

Reply:
fixed